### DISCUSSION TITLE
Sunbird RC Important Links

### DISCUSSION THREAD

**Discussion by jayadinni** [Sunbird-RC/community#270] [General]
Where can you Learn, Develop and Build with Sunbird RC:
- [Documentation](https://docs.sunbirdrc.dev/)
- [Code](https://github.com/Sunbird-RC)

How can you Contribute to Sunbird RC:
- [Discussion](https://github.com/Sunbird-RC/community/discussions)
- [Public community channel on Discord](https://discord.com/channels/928602017746796654/928607291282964490)

**Comment by RanvSing** [comment]
Hello - am trying to use the Credential Issuance demo ...not getting the OTP to continue.
email used is ranvir.singh@educerts.net

**Reply by tejash-jl** [reply]
you can use the default otp `1234`, we have configured a mock otp layer for now.

**Comment by RanvSing** [comment]
Thank you for your response.

**Comment by nikuldev** [comment]
Hello,
I did the sunbird-rc setup on aws-ec2 ubuntu instance.
I want to get token from keycloak which is required for the consumption of the Sunbird APIs.
But I am getting HTTPS required on UI while accessing the admin portal of keycloak.
are there any changes required?


![keycloak](https://user-images.githubusercontent.com/127303530/223699608-368d34ce-8ddd-4f98-8a9c-0fe4b4812f25.PNG)

![Keycloak-network](https://user-images.githubusercontent.com/127303530/223699701-8454dd5a-3c82-4e45-80f8-6f9c7e75ff90.PNG)

**Reply by surendrasinghs** [reply]
Can you please create a fresh/new discussion thread for this under [Questions (FAQs)](https://github.com/orgs/Sunbird-RC/discussions/categories/questions-faqs)? @nikuldev

**Reply by nikuldev** [reply]
@surendrasinghs 
Yes, I will create new discussion for this.
